### DISCUSSION TITLE
Import export stream servers

### DISCUSSION THREAD

**Discussion by resistancegr** [sorayuki/obs-multi-rtmp#370] [Ideas]
Where can i find the file that store the servers? is it possible to import/export all the streaming servers settings and keys? when i update the plugin from obs 27 to obs 30 the list is empty.

**Comment by sorayuki** [comment]
It should load the old configuration during the update......
The logfile will say that how many targets are loaded / saved.
The servers setting is located in the same folder of OBS's profile configuration.
________________________________
From: resistancegr ***@***.***>
Sent: Monday, November 20, 2023 10:53:47 PM
To: sorayuki/obs-multi-rtmp ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [sorayuki/obs-multi-rtmp] Import export steam servers (Discussion #370)


Where can i find the file that store the servers? is it possible to import/export all the streaming servers settings and keys? when i update the plugin from obs 27 to obs 30 the list is empty.

―

**Reply by resistancegr** [reply]
name of the file please?

**Comment by sorayuki** [comment]
obs-multi-rtmp.json
________________________________

**Reply by resistancegr** [reply]
i need the settings file from obs-multi-rtmp-0.3.0.2 .  I cant find any file with this name....(obs-multi-rtmp.json)

search in appdata , program files , programdata with no luck

**Reply by sorayuki** [reply]
For the version before 0.4.0.0, it stores in the profile's ini configuration file, a [obs-multi-rtmp] section with the key said "json".
When updating to 0.4.0.0 or higher, it will be imported into a standalone obs-multi-rtmp.json.
After import the configuration, it will erase the old configuration from ini.

**Reply by resistancegr** [reply]
Thank you 4 the reply's,
i will give one more try to find it. Then i will add the servers manualy (i have 15 servers in my list. not too many).
Try to add import export in the next update. Also if its possible , add ability to move the entrys in the list ...

Thanks and congratulations for the work you did on this plugin so far..... :)
